### ISSUE TITLE
[BUG] [URGENT] Components is not expanding anymore for some bottle

### ISSUE THREAD

**Issue by ghtesting2020** [bottlesdevs/Bottles#1116]
2022.2.28-trento-2

In the latest version for some of my bottles I cannot change the components in Preferences>system.

When you click the arrow to show the dropdown it just moves a couple milimeters but does not show the components anymore!!!!!!!!!!!! For some it works normal, others it expands but is empty. I cannot tell what is different between the bottles. They are all gaming and should behave the same.

**Comment by mirkobrombin** [comment]
Please next time fulfill the bug report template. 

- Which package?
- Are there any interesting logs?

**Comment by ghtesting2020** [comment]
>Which package
Flatpak
>Are there any interesting logs?
No

**Comment by jarrard** [comment]
I noticed this but if you go to another TAB whatever and come back it will be expanded. The same issue happens with toggle buttons not fully showing the nob going to right or left but the color changes.

Bottles UI still needs allot of work.

**Comment by mirkobrombin** [comment]
I did some testing last night with Pop OS 21.10, Silverblue 35, Ubuntu 21.10, Arch Linux and Linux Mint. I have not encountered this problem with any of these distributions.. maybe something on your setup? Maybe not but Bottles is UI/UX-centric and we really take care of details, but in this case I can't reproduce the problem in anyway and I can't plan a fix to "improve the Bottles UI which still needs allot of work" 😔

**Comment by mirkobrombin** [comment]
I'll do another test forcing Bottles to "fails" components downloads and check if the problem came in with no components in list

**Comment by mirkobrombin** [comment]
I'm not able to reproduce. Closing for now, feel free to add more comments and details.

**Comment by jarrard** [comment]
It may be a issue mostly happening on Plasma DE, can't remember it happening on XFCE.

**Comment by mirkobrombin** [comment]
What theme do you see in Bottles, Adwaita?

**Comment by ghtesting2020** [comment]
> It may be a issue mostly happening on Plasma DE, can't remember it happening on XFCE.

Yes that might be it!!!! I use KDE Plasma with Breeze dark no special theme or anything. It is the dark version of default. For Bottles I use the dark version.

**Comment by ghtesting2020** [comment]
@mirkobrombin will this be added to your "testing" list to see if you can reproduce in a KDE Plasma environment or reopen the CR? I am still having this issue with 2022.3.14-trento-3 flatpak.

**Comment by mirkobrombin** [comment]
It is already on my list but I cannot promise to return soon, I am focused on other things. @francescomasala can you test this?

**Comment by ghtesting2020** [comment]
> It's already on my list but I can't promise to get my hands on it soon, I'm focused on other things. @francescomasala can you try it?

this is still an issue in latest 2022.3.28-trento I have to click to expand component and  then to a different tab then back to be able to see it.

**Comment by mirkobrombin** [comment]
![image](https://user-images.githubusercontent.com/9479903/160424371-d6769c7b-2bdf-4018-a491-c14313569234.png)

I've done some tests with KDE and cannot reproduce this.

**Comment by jarrard** [comment]
Yeah its suddenly not happening for me anymore either. Weird. I even use Picom which is a experimental compositor so I dunno what to say. 

Perhaps it only happens under certain conditions or with specific package versions. I am using a 6800XT with Mesa-Git

**Comment by jarrard** [comment]
UPDATE:  The Enable FSR button goes blue but does not shift to right or left until you flip back to other tab.  

There is probably others.
